server/channels/jobs/batch_migration_worker_test.go

```diff
@@ -100,8 +100,8 @@ func TestBatchMigrationWorker(t *testing.T) {
 	}
 
 	t.Run("done after three batches", func(t *testing.T) {
-		th := Setup(t).InitBasic()
-		defer th.TearDown()
+		th := Setup(t).InitBasic(t)
+		defer th.TearDown(t)
```

Comment:
One day, perhaps we can move the `TearDown` into a `t.Cleanup` invoked from inside `th.Setup`.

Reply:
Good suggestion. I did run into issues before when mixing `t.Cleanup` and `defer`. But given that this is a clean migration, it should work fine.